SoftLayer/managers/registration.py

```diff
@@ -0,0 +1,29 @@
+"""
+    SoftLayer.subnet_registration
+    ~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
the `~~~~` need to go to the end of the line above it for the docs to render properly